recipe/meta.yaml

```diff
@@ -12,30 +12,21 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 0
+  number: 1
+  # These are present when the new environment is created
+  # so we have to exempt them from the list of initial files
+  # for conda-build to realize they should be included.
   entry_points:
     - conda = conda.cli:main
     - conda-env = conda_env.cli.main:main
   always_include_files:
```

Comment:
`[unix]` is fine.  Just change the `exec` part to `Scripts` for `[win]`.  Guess we need to update the recipe in the conda repo.  Although it probably already is for 4.4.x.